packages/amplication-server/src/core/organization/sampleAppData.ts

```diff
@@ -0,0 +1,244 @@
+import { Entity, EntityField } from 'src/models';
+
+type EntityFieldData = Omit<
+  EntityField,
+  'id' | 'createdAt' | 'updatedAt' | 'fieldPermanentId'
+>;
+
+type EntityData = Omit<
+  Entity,
+  'id' | 'createdAt' | 'updatedAt' | 'app' | 'appId' | 'fields'
+> & {
+  fields: EntityFieldData[];
+};
+
+export const SAMPLE_APP_COMMIT_MESSAGE = 'Commit Sample App Entities';
+
+//List of entities to be created in the sample app.
+//The list includes references between entities. We use [entityName] notation to allow the bulkCreate operation to map the name to the actual ID.
```

Comment:
Provide an ID for user entity instead of the default created one of Prisma and use it for referencing the model.

Reply:
We should leave the current implementation and not use hard-coded IDs